spec/acceptance/mysql_spec.rb

```diff
@@ -0,0 +1,367 @@
+require 'acceptance_spec_helper'
+
+RSpec.describe 'MySQL sessions and MySQL modules' do
+  include_context 'wait_for_expect'
+
+  RHOST_REGEX = /\d+\.\d+\.\d+\.\d+:\d+/
+
+  TESTS = {
+    mysql: {
+      target: {
+        session_module: "auxiliary/scanner/mysql/mysql_login",
+        type: 'MySQL',
+        platforms: [:linux, :osx, :windows],
+        datastore: {
+          global: {},
+          module: {
+            username: ENV.fetch('MYSQL_USERNAME', 'root'),
+            password: ENV.fetch('MYSQL_ROOT_PASSWORD', 'password'),
+            rhost: ENV.fetch('MYSQL_RHOST', '127.0.0.1'),
+            rport: ENV.fetch('MYSQL_RPORT', '3306'),
+          }
+        }
+      },
+      module_tests: [
```

Comment:
Are there more modules we should be adding here? 🤔 

Or do the other modules require option configuration to run successfully